src/engine/square.ts

```diff
@@ -14,13 +13,9 @@ export default class Square {
         return new Square(row, col);
     }
 
-    public squareAtOffset(offset : Offset){
+    public offset(offset : Offset){
         const {x, y} = offset;
```

Comment:
I think `row` and `column` would be better names for the properties on the `Offset` class as it's more meaningful in this case